src/languages/javascript.coffee

```diff
@@ -15,14 +15,19 @@ module.exports = {
   Supported Grammars
   ###
   grammars: [
-    "JavaScript"
+    "JavaScript",
+    "JavaScript for Automation (JXA)"
   ]
 
   ###
   Supported extensions
   ###
   extensions: [
-    "js"
+    "js",
+	"scpt"
```

Comment:
Is the indentation wrong here?
Travis CI is failing: https://travis-ci.org/Glavin001/atom-beautify/jobs/130276709